vitest.config.mts

```diff
@@ -1,13 +1,14 @@
 /// <reference types="vitest" />
-import { defineConfig } from 'vitest/config'
-import react from '@vitejs/plugin-react'
-import tsconfigPaths from 'vite-tsconfig-paths'
+import { defineConfig } from "vitest/config";
+import react from "@vitejs/plugin-react";
+import tsconfigPaths from "vite-tsconfig-paths";
 
 export default defineConfig({
-    plugins: [tsconfigPaths(), react()],
-    test: {
-        environment: 'jsdom',
-        globals: true,
-        setupFiles: ['./src/test/setup.ts'],
-    },
-});
\ No newline at end of file
+  plugins: [tsconfigPaths(), react()],
+  test: {
+    environment: "jsdom",
+    globals: true,
+    setupFiles: ["./src/test/setup.ts"],
+    exclude: ["**/node_modules/**", "**/dist/**", "**/e2e/**"],
```

Comment:
Avoids running e2e tests from vitest. Using another filename convention to distinguish unit vs e2e would also be a good improvement.